manifests/update.pp

```diff
@@ -2,7 +2,7 @@
 class ca_cert::update {
   include ::ca_cert::params
 
-  if $::osfamily == 'RedHat' {
+  if ($::osfamily == 'RedHat' and versioncmp($::operatingsystemrelease, '7') < 0) {
```

Comment:
I'm not sure how long this module will continue to support puppet 3, but since it is currently marked as supported, I've not switched to the newer non-legacy facts.

Reply:
I don't think we need to support Puppet 3 any longer - I just haven't run across anything that breaks it yet either.

Feel free to break it and update metadata if you want/need to.